### ISSUE TITLE
HTMLElement is not defined

### ISSUE THREAD

**Issue by lvnr** [Stanko/react-plx#25]
Getting this error when trying to use `react-plx` in a `next.js` project.

```
HTMLElement is not defined
ReferenceError: HTMLElement is not defined
    at Object.<anonymous> (/Users/levonarakelyan/projects/mojilytics/landmoji/node_modules/react-plx/lib/Plx.js:757:128)
    at Module._compile (module.js:570:32)
    at Module._compile (/Users/levonarakelyan/projects/mojilytics/landmoji/node_modules/source-map-support/source-map-support.js:492:25)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/levonarakelyan/projects/mojilytics/landmoji/node_modules/react-plx/lib/index.js:8:12)
```

Probably just because there's no `HTMLElement` when it's trying to render it on the server-side.

Thanks a lot @Stanko for a great lib!

**Comment by Stanko** [comment]
Hey @lvnr,
You are probably right, it fails on prop types definitions
`PropTypes.instanceOf(HTMLElement),` 
which gets executed on the server. I'll push a fix in next 30 minutes.

Btw I would love to see animations made using Plx, so if you can please share your work.

Cheers!

**Comment by lvnr** [comment]
@Stanko Thanks a lot for such a quick reply!

Will absolutely send the page your way as soon as it's up! 🙂

**Comment by Stanko** [comment]
Thank you! 🎉 

I just published `v1.0.4` which should take care of your issue. Please let me know if it works for you, so we can close the issue.

Commit: https://github.com/Stanko/react-plx/commit/ac7d713383a715a2bac3112378bd0effcf0b2373

**Comment by lvnr** [comment]
It works @Stanko!

Thanks a lot!
